Modified to support multiple extensions. (#14)
dashbitco/nimble_publisher: README.md

  * `:build` - the name of the module that will build each entry

>   * `:from` - a wildcard pattern where to find all entries. use `Earmark` 
>     to convert to HTML only if the file extension is .md or .markdown. 
>     Other extensions will not be converted.

Reviewer: ```suggestion
  * `:from` - a wildcard pattern where to find all entries. Files with the
    `.md` or `.markdown` extension will be converted to Markdown with
    `Earmark`. Other files will be kept as is.
```
